<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videoDesignerWindow</class>
 <widget class="QMainWindow" name="videoDesignerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>videoDeisgnerWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#videoDesignerWindow
{
	background-color:rgb(53,53,53);	
}
#close_button
{
	background-color:rgb(68,68,68);	
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="VideoWidget" name="video_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>400</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="open_button">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QSlider" name="video_1_slider">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>420</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>亮度（0~100）：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>450</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>对比度（1.0~3.0）：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="bright">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="contrast">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>450</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>3.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>380</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="VideoWidget" name="video_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>400</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="_export">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>380</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>导出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>490</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>图像旋转：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="rotate_combo">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>490</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>90</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>180</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>270</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videowidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>video_1_slider</sender>
   <signal>sliderPressed()</signal>
   <receiver>videoDesignerWindow</receiver>
   <slot>SliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>-48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>video_1_slider</sender>
   <signal>sliderReleased()</signal>
   <receiver>videoDesignerWindow</receiver>
   <slot>SliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>-23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>video_1_slider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>videoDesignerWindow</receiver>
   <slot>SetPos(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>-112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>videoDesignerWindow</receiver>
   <slot>Set()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>691</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>687</x>
     <y>-23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_export</sender>
   <signal>clicked()</signal>
   <receiver>videoDesignerWindow</receiver>
   <slot>Export()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>play</sender>
   <signal>clicked()</signal>
   <receiver>videoDesignerWindow</receiver>
   <slot>Play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SliderPress()</slot>
  <slot>SliderRelease()</slot>
  <slot>SetPos(int)</slot>
  <slot>Set()</slot>
  <slot>Export()</slot>
  <slot>Play()</slot>
 </slots>
</ui>
